libs/utils/utils.js

```diff
@@ -332,6 +332,7 @@ function decorateLinks(el) {
   return [...anchors].reduce((rdx, a) => {
     a.href = makeRelative(a.href);
     decorateSVG(a);
+    if (a.href.includes('#_blank')) a.setAttribute('target', '_blank');
```

Comment:
Should the `#_blank` be removed from the link?

Reply:
It wasn't specified, but it's a good idea. I'll remove it